mindocr/data/predict_dataset.py

```diff
@@ -1,81 +1,73 @@
-"""
-Inference dataset class
-"""
 import os
 import random
 from typing import List, Union
 
+import numpy as np
+
 from .base_dataset import BaseDataset
-from .transforms.transforms_factory import create_transforms, run_transforms
 
 __all__ = ["PredictDataset"]
 
 
 class PredictDataset(BaseDataset):
     """
+    General dataset for parallel online prediction pipeline
+
+    Args:
+        data_dir (str):  directory to the image data
+        sample_ratio (Union[float, List[float]]): sample ratios for the data items in label files
+        shuffle(bool): shuffle samples within dataset. Useful only when `sample_ratio` is set below 1. (Default: False).
+
+    Returns:
+        data (tuple): Depending on the transform pipeline, __get_item__ returns a tuple for the specified data item.
+        You can specify the `output_columns` arg to order the output data for dataloader.
+
     Notes:
-    1. The data file structure should be like
-        ├── img_dir
-        │     ├── 000001.jpg
-        │     ├── 000002.jpg
-        │     ├── {image_file_name}
+        1. The data file structure should be like
+            ├── data_dir
+            │     ├── 000001.jpg
+            │     ├── 000002.jpg
+            │     ├── {image_file_name}
     """
 
     def __init__(
         self,
-        # is_train: bool = False,
-        dataset_root: str = "",
-        data_dir: str = "",
+        data_dir: Union[str, List[str]] = None,
         sample_ratio: Union[List, float] = 1.0,
-        shuffle: bool = None,
-        transform_pipeline: List[dict] = None,
-        output_columns: List[str] = None,
+        shuffle: bool = False,
```

Comment:
I think `sample_ratio` and `shuffle` are not needed in `PredictDataset`.

Reply:
`shuffle` is removed. `sample_ratio` is kept, for the case that users want to quickly see the inference results for checking whether his model is good but there are too much images in his dataset folder.